proto/anki/config.proto

```diff
@@ -47,6 +47,8 @@ message ConfigKey {
     RESTORE_POSITION_REVIEWER = 20;
     RESET_COUNTS_BROWSER = 21;
     RESET_COUNTS_REVIEWER = 22;
+    RANDOM_ORDER_REPOSITION = 23;
+    SHIFT_POSITION_OF_EXISTING_CARDS = 24;
```

Comment:
Technically this is not necessary, as the frontend doesn't need to read or write to these directly, but no harm to include them either.